libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -632,6 +679,7 @@ def _astream(
     ) -> AsyncIterator[ChatGenerationChunk]:
         raise NotImplementedError()
 
+    @deprecated("0.1.0", alternative="invoke")
     def __call__(
```

Comment:
shouldn't we be deprecating acall too?

Reply:
dont think BaseChatModel implements acall